packages/core/src/utils/__tests__/security.spec.ts

```diff
@@ -142,9 +145,9 @@ describe('validateSecurity', () => {
       it('does not add info to WWW-Authenticate header', () => {
         assertSome(
           validateSecurity(
-            { tags: [] },
+            { headers: [] },
```

Comment:
ha, interesting :)

Reply:
neither of these was correct, `tags` was simply not used and `headers` ought to be an object, not an array ;)